cves/CVE-2013-0847.yml

```diff
@@ -169,10 +168,10 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: 'Mateusz "j00ru" Jurczyk and Gynvael Coldwind'
```

Comment:
How exactly did these two discover the vulnerability? Does any information pertaining to that exist?

Reply:
Unfortunately I cannot find anything about how they found the vulnerability. However, I've noticed that they have fixed a lot of out of bound pointer vulnerabilities.